authentication_v2/src/lib/forgotPassword/ForgotPassword.js

```diff
@@ -31,13 +31,19 @@ ForgotPassword.prototype.postForgot = function(req, res) {
     var responseOkMessage = "We've sent an email with the instructions for recover your password";
 
     userData.username = req.body.username;
+    userData.lang = req.body.lang;
```

Comment:
Would be nice to control more this input, we can't trust what we are given by the client and this var is used to build a path later on.